packages/payment-detection/src/erc777/superfluid-retriever.ts

```diff
@@ -66,8 +66,8 @@ export class SuperFluidInfoRetriever {
         oldFlowRate: streamEvents[streamEvents.length - 1].flowRate,
         flowRate: 0,
         timestamp: Utils.getCurrentTimestampInSecond(),
-        blockNumber: null,
-        transactionHash: null,
+        blockNumber: parseInt(streamEvents[streamEvents.length - 1].blockNumber),
```

Comment:
Can you please simplify all the references to `streamEvents[streamEvents.length - 1]` ?

Reply:
Too late for this time but I will do it when I will do the next version of payment-processor for SuperFluid.